src/templates_parser.adb

```diff
@@ -2791,7 +2791,7 @@ package body Templates_Parser is
             Offset := 2;
          end if;
 
-         return Last /= 0
+         return Last >= First + Stmt'Length - 1
```

Comment:
I think it should be: "Last >= First + Stmt'Length - 1 - Offset" no?

Reply:
The Offset need only for comparision extended macros without ending @@, The ending @@ have to be anyway. So the line length have to be not less then minimum macros size.